flagext/bytes_test.go

```diff
@@ -0,0 +1,127 @@
+package flagext
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func Test_Bytes_String(t *testing.T) {
+	for _, tcase := range []struct {
+		value uint64
+		str   string
+	}{
+		{
+			value: 1024 * 1024 * 1024,
+			str:   "1GiB",
+		},
+		{
+			value: 1024 * 1024,
+			str:   "1MiB",
+		},
+		{
+			value: 1024,
+			str:   "1KiB",
+		},
+	} {
+		b := Bytes(tcase.value)
+		require.Equal(t, tcase.str, b.String())
+	}
+}
+
+func Test_Bytes_Set(t *testing.T) {
+	for _, tcase := range []struct {
```

Comment:
[nit] Given `str` is the input value, I'd put it first, as then the test cases read "this is the input, and this is the expected output"